src/main/webapp/courseLoader.js

```diff
@@ -1,127 +1,200 @@
-        legend: {position: "none"},
-        vAxis: {title: "# Students"},
-        hAxis: {title: "Difficulty"},
-        histogram: {
-          hideBucketItems: true
-        }
-      };
-    const diffChart = new google.visualization.Histogram(document.getElementById("diff-chart"));
-    diffChart.draw(diffData, diffOptions);
-
-
-    const tempTermPerceptionList = dataObject.termPerceptionList;
-    const termPerceptionList = [["Term Perception"], /* dummyPerceptionRating */ [11], /* dummyPerceptionRating */ [5]].concat(tempTermPerceptionList);
-    const termPerceptionData = new google.visualization.arrayToDataTable(termPerceptionList);
-    const termPerceptionOptions = {
-        title: "Perception of Term Reviews",
-        legend: {position: "none"},
-        vAxis: {title: "Perception"},
-        hAxis: {title: "Comment Quantity"},
-        histogram: {
-          hideBucketItems: true
-       }
-    };
-    const termPerceptionChart = new google.visualization.Histogram(document.getElementById("term-chart"));
-    termPerceptionChart.draw(termPerceptionData, termPerceptionOptions);
-
-
-    const tempProfPerceptionList = dataObject.termPerceptionList;
-    const profPerceptionList = [["Professor Perception"], /* dummyPerceptionRating */ [2], /* dummyPerceptionRating */ [9]].concat(tempProfPerceptionList);
-    const profPerceptionData = new google.visualization.arrayToDataTable(profPerceptionList);
-    const profPerceptionOptions = {
-        title: "Perception of Professor Reviews",
-        legend: {position: "none"},
-        vAxis: {title: "Perception"},
-        hAxis: {title: "Comment Quantity"},
-        histogram: {
-          hideBucketItems: true
-        }
-      };
-    const profPerceptionChart = new google.visualization.Histogram(document.getElementById("prof-chart"));
-    profPerceptionChart.draw(profPerceptionData, profPerceptionOptions);
+document.addEventListener("DOMContentLoaded", function () {
+  populateData();
+});
+
+function createComment(commentText, flag) {
+  const commentContainer = flag
+    ? document.getElementById("course-comments")
+    : document.getElementById("prof-comments");
+
+  const commentWrapper = document.createElement("div");
+  commentWrapper.setAttribute("class", "media text-muted pt-3");
+
+  const commentUser = document.createElement("strong");
+  commentUser.setAttribute("class", "d-block text-gray-dark");
+  commentUser.innerHTML = "@username";
+
+  const commentBody = document.createElement("p");
+  commentBody.setAttribute(
+    "class",
+    "media-body pb-3 mb-0 small lh-125 border-bottom border-gray"
+  );
+  commentBody.innerHTML = commentText;
+
+  commentBody.insertAdjacentElement("afterbegin", commentUser);
+  commentWrapper.appendChild(commentBody);
+  commentContainer.appendChild(commentWrapper);
 }
 
+function loadComments(commentList, flag) {
+  for (let comment of commentList) {
+    createComment(comment, flag);
+  }
+}
+
+function makeGraphs(dataObject) {
```

Comment:
Not thrilled with the "dataObject" name. It does tell you the structure (object) and that it contains data, but absolutely nothing about that data.
termData or termDataObject would be better imo.